src/vector/feature.rs

```diff
@@ -749,10 +763,10 @@ impl FieldValue {
     }
 
     /// Interpret the value as `Date`. Returns `None` if the value is something else.
-    pub fn into_date(self) -> Option<Date<FixedOffset>> {
+    pub fn into_date(self) -> Option<NaiveDate> {
         match self {
             FieldValue::DateValue(rv) => Some(rv),
-            FieldValue::DateTimeValue(rv) => Some(rv.date()),
+            FieldValue::DateTimeValue(rv) => Some(rv.date_naive()),
```

Comment:
Now that these are fixed via #336 & #339, can you rebase (or merge) primary focus of PR against `master`?

Reply:
Merged `master` into PR 341 branch.  Can squash when merging 341.